alica_tests/etc/conditions/ConditionRepository.cnd

```diff
@@ -581,6 +589,38 @@
       "libraryName": "alica-tests",
       "name": "IsAnyChildStatusSuccess"
     },
+    {
+      "blackboard": [
+        {
+          "access": "input",
+          "comment": "",
+          "id": 1999642097123177760,
+          "key": "expected",
+          "type": "int64"
+        }
+      ],
+      "comment": "",
+      "conditionRepository": 0,
+      "id": 3642911512363445478,
+      "libraryName": "alica-tests",
+      "name": "ExecOrderVectorSizeSmaller"
```

Comment:
I suppose this is unused? Please remove any unused conditions & their references in the code